desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -1800,6 +1805,11 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
       self.interpreter.subscribe(function(val) {
         self.sourceType(val);
         wizard.destination.sourceType(val);
+        for (let i = 0; i < self.interpreters().length; i++) {
```

Comment:
nit: avoid for loop and use e.g. forEach/filter js functions (shorter and cleaner)

Reply:
PS: why not using the val argument actually?

Reply:
We can't use "val" because when we enable the connector (i.e. enable_connector = true) then val is a numeric value which is id and it can be different in each system. So we need a dialect name that is fixed.

Reply:
done